lasagne/tests/test_nonlinearities.py

```diff
@@ -55,6 +72,16 @@ def test_nonlinearity(self, nonlinearity):
         elif nonlinearity == 'scaled_tanh_p':
             from lasagne.nonlinearities import ScaledTanH
             theano_nonlinearity = ScaledTanH(scale_in=0.5, scale_out=2.27)
+        elif nonlinearity.startswith('selu'):
+            from lasagne.nonlinearities import SELU, selu
+            if nonlinearity == 'selu':
+                theano_nonlinearity = SELU(scale=1, scale_neg=1)
```

Comment:
I'd even remove the `scale=1, scale_neg=1` here, so the test also ensures nobody can change the default values without us noticing.